TLM/TLM/packages.config

```diff
@@ -1,10 +1,5 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.CodeAnalysis.FxCopAnalyzers" version="2.9.7" targetFramework="net35" developmentDependency="true" />
```

Comment:
This file should not be modified, you essentially removed analyzers.

Reply:
You have commented on the wrong diff. I accidentally added unwanted code to packages.config and then removed it. So all in all it has not changed.